def onError(err):
    print("Something bad happened: " + err)
